docs/customization/api.rst

```diff
@@ -95,4 +105,362 @@ The next step is to modify the security configuration in ``config/packages/secur
+
+.. code-block:: javascript
+
+    {
+        //...
+        "id": 123,
+        "code": "product_code",
+        "variants": [
+            "/api/v2/shop/product-variants/product-variant-0",
+        ],
+        "translations": {
+            "en_US": {
+              "@id": "/api/v2/shop/product-translations/123",
+              "@type": "ProductTranslation",
+              "id": 123,
+              "name": "product name",
+              "slug": "product-name"
+        }
+    }
+
+Then let's say you want to remove ``translations``.
+
+Utilising serialization groups to remove fields might be quite tricky as Symfony combines all of the serialization files into one.
+The easiest solution to remove the field is to create a new serialization group and use it for fields you want to have and declare this group in the endpoint.
+
+First let's add the ``config/api_platform/Product.xml`` configuration file. See ``How to add an additional endpoint?`` for more information.
+Then let's modify the endpoint. For this example i will use GET item in shop, but you can also create some custom endpoint:
+
+.. code-block:: xml
+
+    <!--...-->
+    <itemOperation name="shop_get">
+        <attribute name="method">GET</attribute>
+        <attribute name="path">/shop/products/{code}</attribute>
+        <attribute name="openapi_context">
+            <attribute name="summary">Use code to retrieve a product resource.</attribute>
+        </attribute>
+        <attribute name="normalization_context">
+            <attribute name="groups">shop:product:read</attribute>
+        </attribute>
+    </itemOperation>
+    <!--...-->
+
+then let's change the serialization group in ``normalization_context`` attribute to `shop:product:custom_read`:
+
+.. code-block:: xml
+
+    <!--...-->
+    <attribute name="normalization_context">
+        <attribute name="groups">shop:product:custom_read</attribute>
+    </attribute>
+    <!--...-->
+
+Now we need to modify the file ``config/serialization/Product.xml`` and add this custom serialization group to fields we want to show:
+
+.. code-block:: xml
+
+    <!--...-->
+    <attribute name="updatedAt">
+            <group>admin:product:read</group>
+    </attribute>
+    <attribute name="translations">
+        <group>admin:product:create</group>
+        <group>admin:product:read</group>
+        <group>admin:product:update</group>
+        <group>shop:product:read</group>
+    </attribute>
+    <attribute name="mainTaxon">
+        <group>admin:product:create</group>
+        <group>admin:product:read</group>
+        <group>admin:product:update</group>
+        <group>shop:product:read</group>
+        <group>shop:product:custom_read</group>
+    </attribute>
+    <!--...-->
+
+.. note::
+
+    In example the ``translations`` doesn't have the new group ``shop:product:custom_read`` so it won't be shown by that endpoint.
+    The rest of the fields that we want to show have the new serialization group declared.
```

Comment:
```suggestion
```

Reply:
I would leave this section (with some changes) so it would be better to understand what is happening.